etcddb/etcddb_conf.go

```diff
@@ -84,5 +88,66 @@ func GetEtcdServerConf(vip *viper.Viper) (conf *EtcdServerConf, err error) {
 	conf.Enabled = true
 
 	err = vip.UnmarshalKey(key, conf)
+
+	if err != nil {
+		return
+	}
+
+	err = initEtcdLogger(conf)
+
+	return
+}
+
+// capnslog to logrus formatter implementation
+// with methods Format and Flush
+type capnslogToLogrusLogFormatter struct {
+}
+
+func (formatter *capnslogToLogrusLogFormatter) Format(pkg string, level capnslog.LogLevel,
+	depth int, entries ...interface{}) {
+
+	l := log.WithFields(log.Fields{
+		"pkg": pkg,
+		"msg": fmt.Sprint(entries...),
```

Comment:
It seems more logical to format entries into the string and use as log message, for instance:
```go
msg := fmt.Sprintf(entries...)
...
l.Info(msg)
```